data/fedora-update.yaml

```diff
@@ -45,13 +45,12 @@ kernel-tests:
     status: in-progress
     links:
         repo: https://git.fedorahosted.org/cgit/kernel-tests.git
+        bug: https://bugzilla.redhat.com/show_bug.cgi?id=1282165
```

Comment:
That's the wrong bug; I changed it to https://bugzilla.redhat.com/show_bug.cgi?id=1293015